astropy/coordinates/earth.py

```diff
@@ -655,22 +655,29 @@ def to_geocentric(self):
         """Convert to a tuple with X, Y, and Z as quantities."""
         return (self.x, self.y, self.z)
 
-    def get_itrs(self, obstime=None):
+    def get_itrs(self, obstime=None, location=None):
         """
         Generates an `~astropy.coordinates.ITRS` object with the location of
-        this object at the requested ``obstime``.
+        this object at the requested ``obstime``, relative to the given
+        ``location`` (if provided).
 
         Parameters
         ----------
         obstime : `~astropy.time.Time` or None
             The ``obstime`` to apply to the new `~astropy.coordinates.ITRS`, or
             if None, the default ``obstime`` will be used.
+        location : `~astropy.coordinates.EarthLocation` or None
+            The observer's location, as an `~astropy.coordinates.EarthLocation`
+            object. If None (the default), the geocentric ITRS coordinates will be
```

Comment:
With the above, I think this can become,
```
A possible observer's location, for a topocentric ITRS position.
If not given (default), a geocentric ITRS object will be created.
```